### DISCUSSION TITLE
LPCC features

### DISCUSSION THREAD

**Discussion by seghersmatthias** [jsingh811/pyAudioProcessing#40] [Ideas]
Hi

It would be awesome if you could add LPC and the corresponding LPCC features. 
Is this on the timeline? 

Regards,
Matthias

**Comment by jsingh811** [comment]
Hello @seghersmatthias
Thank you for bringing it up. It has been on the list of TODOs for a while. It is good to see the interest of the community on this feature.
I have created a ticket for this new feature https://github.com/jsingh811/pyAudioProcessing/issues/41. Will aim to take it up in the coming month. Also, any contributions to the project are always welcome.

Best,
Jyotika
